misc/Kconfig

```diff
@@ -44,9 +44,12 @@ endif
 config TEXT_SECTION_OFFSET
 	hex "TEXT section offset"
 	default 0
+	default 0x200 if BOOTLOADER_MCUBOOT
 	help
 	  This sets and offset before the text section and allows booting images
 	  with custom headers that are inserted at the beginning of the image.
+	  If the application is to be booted by MCUboot, a default suitable
```

Comment:
While you're in there, the help text starts off with a typo:  ``This sets an offset``

Reply:
No longer relevant since the text now reads otherwise.